svc/FetchService.ts

```diff
@@ -452,7 +464,7 @@ export interface FetchOptions {
     autoAbortKey?: string;
 
     /**
-     * True to decode the HTTP response as JSON.
+     * True to decode the HTTP response as JSON. Default false.
```

Comment:
Nit - unless I'm missing something, I don't think we default this to `false` anywhere? Maybe more accurate to say "Unset by default"?

Reply:
Well, its a technical detail that its not defaulted to `false`, and we could potentially remedy that.  But from the users perspective is very much `falsy`!